modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -250,8 +247,10 @@ export default {
       }
 
       button {
+        height: 47px;
```

Comment:
Add this style to AposButton component when the `block` modifier is used

Reply:
Done